tests/test_external_search.py

```diff
@@ -497,10 +497,15 @@ def populate_works(self):
         self.ya_romance.presentation_edition.subtitle = (
             "Modern Fairytale Series, Volume 7"
         )
+        self.ya_romance.presentation_edition.series = "Modern Fairytales"
 
         self.no_age = _work()
         self.no_age.summary_text = "President Barack Obama's election in 2008 energized the United States"
 
+        # Set the series to the empty string rather than None -- this isn't counted
+        # as the book belonging to a series.
+        self.no_age.presentation_edition.series = ""
```

Comment:
It's not very clear to me what should happen in this case: according to condition in `external_search` a new condition matching empty series will be added to the chain. Isn't it what `series=False` should do?

Reply:
As background, this test is creating a pretty large number of books, indexing them in Elasticsearch, and then seeing which books show up in the search results when different search queries and filters are provided.

Setting series=False in the Filter constructor will add two extra clauses to the filter:

1. The 'series' field on a book must be indexed: `Exists(field="series")`
2. The indexed value must not be identical to the empty string: `Bool(must_not=[Term(**{"series.keyword": ""})])`

This sets up a test of that second clause. Elasticsearch indexes a field only if the value is not null, and it treats the empty string as a non-null value. Without that second clause, the `no_age` book will incorrectly show up in the test run with the filter `Filter(series=True)`, because Elasticsearch will notice that the 'series' field was indexed.

Reply:
If I did add support for `series=False` then this same book would we used in a second assertion: we would check that it _does_ show up in a query with `series=False`, even though its `series` field is set to a non-null value.

Reply:
Now it's clear, thank you for the clarification.